cmd/steal.go

```diff
@@ -113,3 +94,31 @@ func bufferer(buf *bytes.Buffer, rowChan chan []*database.Cell, done chan bool,
 		}
 	}
 }
+
+func waitGroupBufferer(inputConn *sql.DB, tables []string, dumper *database.MySQLDumper, out chan []*database.Cell, done chan bool, wg *sync.WaitGroup) []*bytes.Buffer {
+
+	var tableBuffers []*bytes.Buffer
+	for _, table := range tables {
+		columns, err := dumper.GetColumns(table)
+		buf := bytes.NewBufferString(fmt.Sprintf("\nINSERT INTO `%s` (%s) VALUES", table, strings.Join(columns, ", ")))
+
+		wg.Add(1)
+		go bufferer(buf, out, done, wg)
+
+		anonymiser := database.NewMySQLAnonymiser(inputConn)
```

Comment:
We're better to create the anonymiser outside and pass it in as a param, rather than passing in the db connection

Reply:
Sounds good. Done.